app/src/main/java/com/nilhcem/blenamebadge/adapter/DrawableAdapter.kt

```diff
@@ -45,6 +45,10 @@ class DrawableAdapter(private val context: Context, private val list: List<Drawa
         return if (selectedPosition == -1) -1 else list[selectedPosition]
     }
 
+    fun getDefaultItem(): DrawableInfo {
+        return list[9]
```

Comment:
9 is a magic number. What if the list is modified or re-arranged or has less members or new members are added